src/main/java/net/snowflake/client/log/SFClientConfig.java

```diff
@@ -0,0 +1,81 @@
+package net.snowflake.client.log;
```

Comment:
Can you put this class outside of log package? as in future this can be used for all other properties and not only for log properties.

Reply:
ah, good catch! Will update it shortly.